sectorbuilder.go

```diff
@@ -445,30 +444,22 @@ func sizeFromConfig(cfg Config) (abi.SectorSize, error) {
 
 func sizeFromProofType(p abi.RegisteredProof) (abi.SectorSize, error) {
 	switch p {
-	case abi.RegisteredProof_WinStackedDRG32GiBSeal:
-		return 1 << 35, nil
-	case abi.RegisteredProof_WinStackedDRG32GiBPoSt:
-		return 1 << 35, nil
 	case abi.RegisteredProof_StackedDRG32GiBSeal:
-		return 1 << 35, nil
+		return 32 << 30, nil
```

Comment:
bug!

Reply:
No, it should be 30 (2^30 = 1GiB)

Reply:
(the post one is wrong)

Reply:
@magik6k 

> No, it should be 30 (2^30 = 1GiB)

Ah, OK. My bad.

We might as well just use the same expressions that exist in the [source-of-truth](https://github.com/filecoin-project/rust-fil-proofs/blob/master/filecoin-proofs/src/constants.rs#L11). Thoughts, @magik6k ?